aedanjames/relational_rails
# User story 2 aedan

### User Story 2 Complete-Organizations 
## Features/Testing
- Established ActiveRecord one to many relationship between organization and fighters. 
- Added testing functionality gems to GEMFILE
- Added organization and fighter seeds.
- Created feature/model tests for organizations. 
- Created model test for fighters
- Refactored 'organizations/:id' path to singular 'organization/:id' path
- ## To Do 
- Add .destroy to top of db/seeds file for both fighters and organizations. As shown in class today.
- Review User Story 3 to ensure functionality is present/write tests
- Add more feature tests
- Evaluate whether OrganizationFighters controller should be implemented.
